### PR TITLE
Add TTYs to make sure a tty is available always

### PR DESCRIPTION
**Pull request by SKumarDas**
Currently there is only one tty which is taken up by the UI application.
Because of which debugging becomes difficult. This code change will enable having multiple
TTYs (4 TTYs as of now).

Signed-off-by: Sapan Das <SKumarDas@luxoft.com>

### CODE REVIEW

**Comment by GordanM** [comment]
Nice job! 

Just a small note on the commit log: it would be better to use the passive voice, instead of "we have" use "there is". Also, the 1st and the 3rd sentence of the log are enough there is no need to write a lot and lose the core of the message. This is a good excercise because when contributing to the software-factory it would be good to be concise and not add a lot of text that doesn't give value. Do you agree with this?

File: recipes-core/systemd/systemd_%.bbappend [hunk 1]
```diff
@@ -12,6 +12,12 @@ do_install_append() {
     	echo "Address=${STATIC_IP_ADDRESS}/24" >> ${D}${sysconfdir}/systemd/network/10-pelux-ethernet.network
     fi
     ln -s /run/systemd/resolve/resolv.conf ${D}${sysconfdir}/resolv.conf
+
+    # Create mutiple consoles
+    tmp="${TTY_LIST}"
+    for ttydev in $tmp ; do
```
Comment (tobsan) [inline recipes-core/systemd/systemd_%.bbappend]: It should be possible to do just `for ttydev in "${TTY_LIST}"`, right?

File: recipes-core/systemd/systemd_%.bbappend [hunk 2]
```diff
@@ -12,6 +12,12 @@ do_install_append() {
...
+        ln -sf ${systemd_unitdir}/system/getty@.service ${D}${sysconfdir}/systemd/system/getty.target.wants/getty@$ttydev.service
```
Comment (tobsan) [inline recipes-core/systemd/systemd_%.bbappend]: You have probably tried this, but just want to check if this is the preferred place to put those service files, as opposed to, for example, `multi-user.target.wants`.

Comment (SKumarDas) [reply recipes-core/systemd/systemd_%.bbappend]: These service files will get created as part of getty.target. So whichever run level includes getty.target, These service files will be created.

File: recipes-core/systemd/systemd_%.bbappend (hunk 1 above)
Comment (SKumarDas) [reply recipes-core/systemd/systemd_%.bbappend]: To make it look clean I have done this way, as in other .bb file systemd-serialgetty.bb  they have done this way.

**Comment by tobsan** [comment]
@SKumarDas you mentioned yesterday that you had issues with the symlinks not being created unless you did a fresh build, is that still the case?

File: recipes-core/systemd/systemd_%.bbappend (hunk 2 above)
Comment (SKumarDas) [reply recipes-core/systemd/systemd_%.bbappend]: @GordanM  Yes still now my changes on creating symlink do not reflect in subsequent build.

Comment (tobsan) [reply recipes-core/systemd/systemd_%.bbappend]: I would try to do a ``rm -rf ${D}${sysconfdir}/systemd/system/getty.target.wants/`` before the symlink step then.

Comment (SKumarDas) [reply recipes-core/systemd/systemd_%.bbappend]: @GordanM  @tobsan , The image is containing my changes after making change to creating symlink. It is just that the dd command that I use to write the image to pendrive is not writing the newly introduced symlinks. If I am formatting the pendrive and then using dd cmd to write the image, it is actually showing the newly introduced symlinks as well.

Comment (tobsan) [reply recipes-core/systemd/systemd_%.bbappend]: That's strange. Can you try to add `FILES_${PN} += ${sysconfdir}/systemd/system/getty.target.wants` and see if that changes anything?

**Comment by GordanM** [comment]
@SKumarDas please use the workflow stated at http://pelux.io/software-factory/master/swf-blueprint/docs/articles/workflow/git-workflow.html and address my comments before merging this PR.

**Comment by tobsan** [comment]
@GordanM can you approve this?

**Pull request merged**